src/components/Image/Image.tsx

```diff
@@ -0,0 +1,39 @@
+interface ImageProps {
+  src: string;
+  alt: string;
+  context:
+    | "horizontal"
+    | "gallery"
+    | "default"
+    | "equipment"
+    | "staffing"
+    | "modal";
+}
+
+export const Image = ({ src, alt, context }: ImageProps) => {
+  return (
+    <img
+      src={src}
+      alt={alt}
+      className={`object-cover rounded-[40px] ${
+        context === "horizontal"
```

Comment:
I agree with Ruven here on the idea of not abusing the ternary operator. Although I am not sure that the switch statement would look much cleaner. I found the solution attached below to look more aesthetically pleasing. This is just preference though. Besides that, I love the fact that there are so many options, especially inside of storybook, I feel this is an amazing way to create user friendly UI especially for whoever will be managing this at the company, well done! 
<img width="781" alt="image" src="https://github.com/user-attachments/assets/883db225-3d72-4e75-98c6-9c5ea38347b1">
I also attached what a switch statement might look like instead 
<img width="781" alt="image" src="https://github.com/user-attachments/assets/f02f760d-5909-4bdf-807d-9c0369686748">

Reply:
I agree with @XDRO and @RPinkha here. I'm as big a fan of ternaries as there are out there, but it's meant to simplify the reading of code, and this looks a bit unweildy unfortunately. Since we have so many options, I would suggest first creating something like a `className` variable above the `return` statement, iterate through the options available to create the final class name you need, and then you can just plug it in here.

Reply:
Great, thank you all for the suggestions! I did my best to implement the suggestion from @tomershvueli Is this what you were thinking I should do? I did end up using switch like @RPinkha had recommended at the beginning as well. Thank you for any further feedback.